locale/en/LC_MESSAGES/django.po

```diff
@@ -17,52 +17,50 @@ msgstr ""
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 
-#: build/lib/medical/schema.py:72 build/lib/medical/schema.py:83
-#: build/lib/medical/schema.py:95 medical/schema.py:72 medical/schema.py:83
-#: medical/schema.py:95 medical/gql_mutations.py:147
-#: medical/gql_mutations.py:210 medical/gql_mutations.py:280
-#: medical/schema.py:97 medical/schema.py:110 medical/schema.py:131
-#: medical/schema.py:153 medical/schema.py:173
+#: medical/gql_mutations.py:162 medical/gql_mutations.py:225
+#: medical/gql_mutations.py:295 medical/schema.py:108 medical/schema.py:121
+#: medical/schema.py:143 medical/schema.py:165 medical/schema.py:186
+#: medical/schema.py:204 medical/schema.py:210
 msgid "unauthorized"
 msgstr "User not authorized for this operation"
 
-#: medical/gql_mutations.py:130
+#: medical/gql_mutations.py:152
 msgid "Code already exists."
```

Comment:
This should be changed for some proper id like mutation.code_already_exists

Reply:
Will do that